### ISSUE TITLE
Grammar shift / reduce conflict near `Constructor`

### ISSUE THREAD

**Issue by RomainTHD** [RomainTHD-UniStuff/JLC#2]

**Comment by RomainTHD** [comment]
```
Warning : *** Shift/Reduce conflict found in state #63
  between Expr6 ::= _SYMB_37 Constructor (*) 
  and     Constructor ::= Constructor (*) _SYMB_10 Expr _SYMB_11 
  under symbol _SYMB_10
  Resolved in favor of shifting.
```

**Comment by RomainTHD** [comment]
Fixed by rewriting the grammar in 8cb4607de3e660d30d10e61c63cde81b473b311d
